domains/grusio.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "mattiagrusio",
+        "email": "mattiagrusio@gmail.com"
+    },
+    "record": {
+        "A": ["2.231.198.113"]
```

Comment:
I cannot reach your site
"2.231.198.113 didn’t send any data."

Reply:
sorry, now should work, can you try again?

Reply:
yeah it works